about.html

```diff
@@ -0,0 +1,80 @@
+            <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="hero" id="hero">
+             <h3>Hello! Online Available Furniture</h3>
+             <h1>Beautifully Designed Furniture</h1>
+             <p class="msg">A website where all kind of furniture is available. Beatifully designed and best furniture is available and it's delivery is also available. 100%% best quality furniture is available at your door step.
+                 You can choose and buy your favourite furniture from our website and we can deliver it to you.
+             </p>
+            <p id="contact">
+               Please contact us via email for any further question about Homifyers online furniture shop
+               <a href="#">admin@homifyers.com</a>
+           </p>
+        </section>
+        <section class="middle">
+            <h2>Homifyers logo</h2>
+            <div class="sprtr"></div>
+            <p class="logo_txt">The logo of Homifyers was decided by its owner in 2021</p>
+            <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+            <h2>See old pictures of our warehouse</h2>
+            <div class="sprtr"></div>
+            <div class="old_pics">
+                <img src="images/old1.jpg" alt="first picture of warehouse">
+                <img src="images/old2.webp" alt="second picture of warehouse">
+            </div>
+        </section>
+        <script src="js/index.js"></script>
+    </main>
+    <footer>
+        <section class="ftr" id="brands">
+            <h2>Brands</h2>
+            <div class="ctsprtr"></div>
+            <ul>
+                <li><img src="images/14-mozilla-logo.png" alt="mozzila"></li>
+                <li><img src="images/15-google-logo.png" alt="google"></li>
+                <li><img src="images/16-naver-logo.png" alt="naver"></li>
+                <li><img src="images/17-daumkakao-logo (3).jfif" alt="daumkakao"></li>
+                <li><img src="images/18-Airbnb-logo.png" alt=""></li>
+            </ul>
+        </section>
+        <section class="last">
+            <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+            <div class="txt">2022 All Righs Reserved</div>
+        </section>
+    </footer>
```

Comment:
- [ ] The footer on the about page should be dark, kindly consider changing the background color to a darker color